# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../vnuml/vnuml-1.8.9-perl-vendor-install.patch
# Copyright (C) 2011 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

Description: Install perl modules into perl's 'vendor' directory

According to http://search.cpan.org/dist/perl/INSTALL#Installation_Directories
this is the right place for distributions which build binary packages of perl
add-on modules.

--- vnuml-1.8.9/configure.ac.orig	2011-05-20 12:47:44.464002358 +0200
+++ vnuml-1.8.9/configure.ac	2011-05-20 12:57:46.228002358 +0200
@@ -40,7 +40,7 @@
 fi
 
 # Now implemented with --with-perl-modules-installdir
-#AC_SUBST(PERL_MODULES_DIR,["`$PERL -e 'use Config;print $Config{installsitelib}'`/VNUML"])
+#AC_SUBST(PERL_MODULES_DIR,["`$PERL -e 'use Config;print $Config{installvendorlib}'`/VNUML"])
 
 dnl required programs
 AC_PROG_LN_S
@@ -243,7 +243,7 @@
 	    AC_HELP_STRING([--with-perl_modules_installroot=DIR],
 	    		   [root directory where perl modules will be installed (default: extracted from Perl output)]),
             PERL_MODULES_INSTALLROOT=$withval,
-            PERL_MODULES_INSTALLROOT=["`$PERL -e 'use Config;print $Config{installsitelib}'`"] )
+            PERL_MODULES_INSTALLROOT=["`$PERL -e 'use Config;print $Config{installvendorlib}'`"] )
 AC_SUBST(PERL_MODULES_INSTALLROOT)
 AC_CHECK_FILE([$PERL_MODULES_INSTALLROOT], AC_SUBST(PERL_MODULES_INSTALLROOT), AC_MSG_ERROR([Invalid Perl modules installroot directory: $VNUML_PERL_MODULES_INSTALLROOT. Please set the appropriate installroot directory using the --with-perl-modules-installroot.]))
 
